checker/totals.go

```diff
@@ -0,0 +1,95 @@
+package checker
+
+import (
+	"encoding/csv"
+	"io"
+	"log"
+	"time"
+)
+
+// DomainTotals compiled aggregated stats across domains.
+// Implements ResultHandler.
+type DomainTotals struct {
+	Time          time.Time
+	Source        string
+	Attempted     int
+	Connected     int // Connected to at least one mx
+	MTASTSTesting []string
+	MTASTSEnforce []string
+}
+
+// HandleDomain adds the result of a single domain scan to aggregated stats.
+func (t *DomainTotals) HandleDomain(r DomainResult) {
```

Comment:
nit: Handle may be vague-- something like "ProcessDomain" or "AddResult" ?

Reply:
I kept it general when naming this fn on the `ResultHandler` interface so it still makes sense when it does something like print the domain result. Maybe I could add a comment noting that it implements the `ResultHandler` interface? Or I can go with `ProcessDomain` if you think it still makes sense in that context.